pages/add/form/get-help-fields.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import validator from './validator';
 import Service from '../../../js/service';
 
-let HelpTypes = React.createClass({
-  getInitialState() {
-    return {
+class HelpTypes extends React.Component {
+  constructor() {
```

Comment:
Important to use `constructor(props) { super(props); ... }` here, if you want to be able to rely on `this.props` in the component at all.